cleanco/classify.py

```diff
@@ -3,59 +3,69 @@
 
 Examples of use:
 
->> # check name for its possible business type(s)
->> classification_sources = typesources()
->> matches("MyCompany Ltd", classification_sources)
-['Limited']
->>
-
->> # check name for its possible jurisdictions, usually countries
->> classification_sources = countrysources()
->> matches("MyCompany Ltd", classification_sources)
-['New Zealand', 'United Kingdom', 'United States of America']
->>
+# check name for its possible business type(s)
+>> matches("MyCompany Ltd", "local_name")
+['Private Limited by Guarantee', 'Private Limited Company', 'Limited Liability Company', 'New Generation Co-operatives']
+
+# check name for its possible countries
+>> matches("MyCompany Ltd", "country")
+['United States of America', 'United Kingdom', 'Canada']
 
 """
 
-from .termdata import terms_by_country, terms_by_type
+from iso20275 import Elf, ElfEntry
 from .clean import strip_tail, normalized
 
 
-def typesources():
-   "business types / abbreviations sorted by length of business type"
-   types = []
-   for business_type in terms_by_type:
-       for item in terms_by_type[business_type]:
-           types.append((business_type, item))
+def classifierList():
+    property_names = [p for p in dir(ElfEntry) if isinstance(
```

Comment:
I wonder if dir() is necessary here or would using __dict__.keys() suffice? Or is this function used somewhere even?

Reply:
It isn't really used anywhere. It is just there for the user to be able to see what their options are in regard to classification. If you think it should be removed in favor of documentation, I'd be for that. However, someone could potentially iterate through that list if they wanted to return all data.

Reply:
I am using __dict__.keys() now. Much cleaner.